src/nbl/video/CVulkanLogicalDevice.h

```diff
@@ -530,7 +530,7 @@ class CVulkanLogicalDevice final : public ILogicalDevice
                     shaderStage, cpushader->getFilepathHint().c_str(), 4u, logger);
 
             spirv = m_physicalDevice->getGLSLCompiler()->compileSPIRVFromGLSL(
```

Comment:
everything up until here would be the same for HLSL and GLSL, also would be to take a SPIR-V optimizer + compiler options struct or something

Reply:
https://github.com/Devsh-Graphics-Programming/Nabla/pull/432/files#r1017795668 dont take a SPIR-V optimizer anymore for `createSpecializedShader`, take it here